Rakefile

```
@@ -68,28 +54,49 @@ parsed_files = PARSER_FILES.map do |parser_file|
     puts "Generating #{parsed_file}..."
     case ext
     when '.ry' # need racc
-      racc = Gem.bin_path 'racc', 'racc'
       rb_file = parser_file.gsub(/\.ry\z/, ".rb")
-      ruby "#{racc} -l -E -o #{rb_file} #{parser_file}"
+      sh "bundle exec racc -l -E -o #{rb_file} #{parser_file}"
```

In general, `sh("command", "arg1", "arg2", ...)` is safer than `sh("command line")`:

```suggestion
      sh "bundle", "exec", "racc", "-l", "-E", "-o", rb_file, parser_file
```